flink-runtime/src/main/java/org/apache/flink/runtime/util/ZooKeeperUtils.java

```diff
@@ -370,11 +370,11 @@ public static String generateZookeeperPath(String root, String namespace) {
 		 * Return the configured {@link ZkClientACLMode}.
 		 *
 		 * @param config The config to parse
-		 * @return Configured ACL mode or {@link HighAvailabilityOptions#ZOOKEEPER_CLIENT_ACL} if not
+		 * @return Configured ACL mode or "open" if not
```

Comment:
This may become outdated; it's better to do something like "or the default defined by {@link HighAvailabilityOptions#ZOOKEEPER_CLIENT_ACL}"